Sources/RInAppMessaging/Utilities/BundleInfo.swift

```diff
@@ -58,8 +58,20 @@ internal extension Bundle {
     static var unitTests: Bundle? {
         .init(identifier: "jp.co.rakuten.inappmessaging.Tests") ?? .init(identifier: "Tests")
     }
+    
+    static var rmcResources: Bundle? {
```

Comment:
@SoumenRautray In order to inject Bundle.main instance in GetConfigRequest and call instance function `rmcResources`, please remove `static`.